src/modules/ekf2/EKF/velocity_fusion.cpp

```diff
@@ -79,12 +82,61 @@ void Ekf::updateVelocityAidSrcStatus(const uint64_t &time_us, const Vector3f &ob
 	aid_src.timestamp_sample = time_us;
 }
 
+void Ekf::fuseBodyVelocity(estimator_aid_source3d_s &aid_src)
+{
+	Vector3f innov;
+	Vector3f innov_var;
+	matrix::Matrix<float, 1, 23UL> H;
```

Comment:
We have an alias for a vector that has the size of the tangent space
```suggestion
	VectorState H;
```

Reply:
First wanted that the equation is consistent with the code, so H would be a 1x23 Vector: K = P * H.T /innov
But I'll change it.